### PR TITLE
processing /images/ URLs in our extension stylesheets

### PR DESCRIPTION
**Pull request by 2is10**
This way we can finally style with background images normally, instead of creating `data:` URLs or using inline styles.

@andrewconner 
cc @martinraison

### CODE REVIEW

**Comment by andrewconner** [comment]
Oh wow, nice. New feature of FF?

File: packrat/bin/build.sh [hunk 1]
```diff
@@ -47,7 +47,9 @@ for d in $(find styles -type d); do
 done
 for f in $(find styles -name '*.css' -not -name 'insulate.css' -not -path 'styles/iframes/*'); do
   # repeat the first class name that occurs in each selector outside of parentheses since repetition is not allowed within :not(...)
-  sed -E -e 's/ *\/\*.*\*\/$//g' -e '/^[^@]*[,{]$/ s/(^|[^(])(\.[a-zA-Z0-9_-]*)/\1\2\2\2/' $f | tee "out/chrome/$f" > "out/firefox/data/$f"
+  css=$(sed -E -e 's/ *\/\*.*\*\/$//g' -e '/^[^@]*[,{]$/ s/(^|[^(])(\.[a-zA-Z0-9_-]*)/\1\2\2\2/' $f)
+  echo "${css//\/images\//chrome-extension://__MSG_@@extension_id__/images/}" > "out/chrome/$f"
```
Comment (2is10) [inline packrat/bin/build.sh:51]: Chrome substitutes our actual extension ID here at the time of stylesheet injection from the `"content_scripts"` section of manifest.json and/or when `chrome.tabs.insertCss` is called.

**Comment by 2is10** [comment]
Firefox has been able to do this with images for years, but still [doesn’t allow iframes](https://bugzilla.mozilla.org/show_bug.cgi?id=792479) to be loaded this way. We had trouble with Chrome because extension IDs are auto-generated and different during iterative development, and the magic `__MSG_@@extension_id__` replaced string solution wasn’t well documented. The two most widely recommended approaches were using JS to insert a `background-image` inline style or use `data:` URLs in stylesheets. There was also a regression in Chrome last year where `chrome-extension:` images loaded slowly (in the same queue as images from the host page), but that was fixed and now Chrome loads them instantly like `data:` images.

**Comment by martinraison** [comment]
great!

**Pull request merged**